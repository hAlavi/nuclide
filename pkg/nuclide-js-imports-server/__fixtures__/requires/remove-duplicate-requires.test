
const {a} = require('Z');
const {b} = require('Z');

import type {A} from 'Z';
import type {B} from 'Z';

require('P');
require('P');

const M = require('M');
const M = require('M');

import type N from 'N';
import type N from 'N';

import typeof O from 'O';
import typeof O from 'O';

const [j] = require('Y');
const [j, k] = require('Y');

import F from 'F';
import F from 'F';

import {g} from 'G';
import {h} from 'G';

const x: A<B, N, O> = a(b, M, j, k, F, g, h);

// @nolint
