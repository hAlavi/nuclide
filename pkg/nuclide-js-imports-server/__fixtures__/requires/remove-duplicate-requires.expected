import type N from 'N';
import typeof O from 'O';
import type {A, B} from 'Z';

require('P');

const M = require('M');
const [j, k] = require('Y');
const {a, b} = require('Z');

import F from 'F';
import {g, h} from 'G';

const x: A<B, N, O> = a(b, M, j, k, F, g, h);

// @nolint
